validator/src/bin/solana-test-validator.rs

```diff
@@ -396,6 +403,9 @@ fn main() {
         IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)),
         faucet_port,
     ));
+    #[cfg(target_family = "windows")]
+    let bpf_jit = false;
```

Comment:
Can you add a comment in the code here that explains why the JIT is forced off for windows